vllm/model_executor/layers/sampler.py

```diff
@@ -3,24 +3,27 @@
 import warnings
 from importlib.util import find_spec
 from math import inf
-from typing import Dict, List, Optional, Tuple
+from typing import Dict, List, Optional, Tuple, Union
 
+import msgspec
 import torch
 import torch.nn as nn
 
+from vllm.spec_decode.metrics import SpecDecodeWorkerMetrics
 from vllm.triton_utils import HAS_TRITON
 
 if HAS_TRITON:
     from vllm.model_executor.layers.ops.sample import sample as sample_triton
 
+from dataclasses import dataclass
+
```

Comment:
Move to the top as dataclass is Python build package.
```python
import warnings
from dataclasses import dataclass
from importlib.util import find_spec
```

Reply:
Fixed